lib/spack/spack/test/cc.py

```diff
@@ -306,10 +306,14 @@ def test_cc_flags(wrapper_environment, wrapper_flags):
         test_args,
         [real_cc]
         + target_args
+        + test_include_paths
+        + ["".join(spack_ldflags)]
```

Comment:
this doesn't seem to be scalable to testing longer sets of LDFLAGS.

Reply:
fixed